src/averageAges.js

```diff
@@ -1,25 +1,27 @@
 'use strict';
 
+function calculateAverageAge(somePeople) {
+  const sumAll = somePeople.reduce((sum, person) => {
+    return sum + (person.died - person.born);
+  }, 0);
+
+  return sumAll / somePeople.length;
+}
+
 /**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = people.filter(person => person.sex === 'm');
```

Comment:
```suggestion
  const men = people.filter(person => person.sex === 'm');
```
Don't use names of data structures of data types in your variables' names